diff -Nur shared-mime-info-0.17/freedesktop.org.xml.in shared-mime-info-0.17.new/freedesktop.org.xml.in
--- shared-mime-info-0.17/freedesktop.org.xml.in	2006-05-23 15:20:59.000000000 +0200
+++ shared-mime-info-0.17.new/freedesktop.org.xml.in	2006-05-23 15:21:30.000000000 +0200
@@ -2375,7 +2375,6 @@
   </mime-type>
   <mime-type type="inode/directory">
     <_comment>folder</_comment>
-    <alias type="x-directory/normal"/>
   </mime-type>
   <mime-type type="inode/fifo">
     <_comment>pipe</_comment>
