diff -urN aaa/shared-mime-info-0.8/configure.in shared-mime-info-0.8/configure.in
--- aaa/shared-mime-info-0.8/configure.in	Wed Jul 24 18:12:09 2002
+++ shared-mime-info-0.8/configure.in	Thu Jul 25 22:54:24 2002
@@ -64,6 +64,6 @@
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
 
-AC_SUBST([mimedir],         ['${datadir}/mime'])
+AC_SUBST([mimedir],         ['${datadir}/mime-info'])
 
 AC_OUTPUT(Makefile)
