--- shared-mime-info-0.14/freedesktop.org.xml.in.dicom	2004-08-31 13:56:42.868986866 +0200
+++ shared-mime-info-0.14/freedesktop.org.xml.in	2004-08-31 14:03:56.637189629 +0200
@@ -58,6 +58,13 @@
     <_comment>Andrew Toolkit inset</_comment>
     <glob pattern="*.ez"/>
   </mime-type>
+  <mime-type type="application/dicom">
+    <_comment>DICOM medical imaging data</_comment>
+    <magic priority="50">
+      <match type="string" value="DICM" offset="128"/>
+    </magic>
+    <glob pattern="*.dcm"/>
+  </mime-type>
   <mime-type type="application/illustrator">
     <_comment>Adobe Illustrator document</_comment>
     <glob pattern="*.ai"/>
